scripts/portstat

```diff
@@ -261,13 +332,15 @@ Examples:
   portstat -d -t test
   portstat
   portstat -r
+  portstat -p 20
 """)
     parser.add_argument('-c', '--clear', action='store_true', help='Copy & clear stats')
     parser.add_argument('-d', '--delete', action='store_true', help='Delete saved stats, either the uid or the specified tag')
     parser.add_argument('-D', '--delete-all', action='store_true', help='Delete all saved stats')
     parser.add_argument('-j', '--json', action='store_true', help='Display in JSON format')
     parser.add_argument('-r', '--raw', action='store_true', help='Raw stats (unmodified output of netstat)')
     parser.add_argument('-t', '--tag', type=str, help='Save stats with name TAG', default=None)
+    parser.add_argument('-p', '--period', type=int, help='Get the portstat within Period seconds', default=0)
```

Comment:
Make the help message clearer. Something like, "Display stats over a specified period (in seconds)."